src/App.js

```diff
@@ -3,15 +3,18 @@ import { useEffect } from "react"
 import { Routes, Route } from "react-router-dom";
 import Mockman from "mockman-js";
 import { Header, Sidebar, SignIn, SignUp } from "components/index"
-import { NotFound, LandingPage, HistoryPage, WatchLaterPage, LikedPage, PlayListPage, VideoPage, AuthorizationPage, RestrictAuth } from "pages/index"
+import { NotFound, LandingPage, HistoryPage, WatchLaterPage, LikedPage, PlayListPage, VideoPage, AuthorizationPage, RestrictAuth, IndividualPlayListPage } from "pages/index"
 import { ToastContainer } from 'react-toastify';
 import 'react-toastify/dist/ReactToastify.css';
 import { useAuth } from 'context/authContext';
+import { usePlayList } from 'context/playListContext';
```

Comment:
Here you can make an index.js in context and then import useAuth and usePlaylist in a single line it would look cleaner.